docs/source/interfaces/ISIS_Reflectometry.rst

```diff
@@ -388,22 +387,93 @@ Hovering over the highlighted run with your cursor will allow you to see why the
+|                               | have the ability to create directories. As the       |
+|                               | naming of files is automatic, the path must also     |
+|                               | point to a directory rather than a file.             |
++-------------------------------+------------------------------------------------------+
+| Prefix                        | The prefix is what is added to the beginning of      |
+|                               | the workspace name to create the file name. No       |
+|                               | underscore or space is added between them so they    |
+|                               | must be manually added.                              |
++-------------------------------+------------------------------------------------------+
+| Filter                        | This can be specified to filter out workspaces       |
+|                               | in the workspace list whose name does not match      |
+|                               | that of the filter text.                             |
++-------------------------------+------------------------------------------------------+
+| Regex                         | Checking this option allows a regular expression     |
+|                               | to be used for filtering workspace names.            |
++-------------------------------+------------------------------------------------------+
+| List Of Workspaces            | The left listbox will contain any workspaces loaded  |
+|                               | into mantid. Double clicking on one will fill        |
+|                               | the right list box with the parameters it contains.  |
+|                               | This listbox supports multi-select in order to       |
+|                               | allow for multiple workspaces to be saved out        |
+|                               | at the same time with the same settings.             |
++-------------------------------+------------------------------------------------------+
+| List Of Logged Parameters     | The right listbox starts out empty, but will fill    |
+|                               | with parameter names when a workspace in the left    |
+|                               | listbox is double clicked. This listbox supports     |
+|                               | multi-select in order to allow for the save output   |
+|                               | to contain multiple parameter notes.                 |
++-------------------------------+------------------------------------------------------+
+| Spectra list                  | Spectra list entry allows you to specify what        |
+|                               | spectra you want in the output file. If this is      |
+|                               | blank, all spectra are included.                     |
++-------------------------------+------------------------------------------------------+
```

Comment:
Also, figure `docs/source/images/ISISReflectometryPolref_save_tab.png` will need to be updated too